code/go/0chain.net/smartcontract/storagesc/config_settigns.go

```diff
@@ -544,6 +662,8 @@ func (conf *Config) get(key Setting) interface{} {
 		return conf.BlockReward.BlobberUsageWeight
 	case ExposeMpt:
 		return conf.ExposeMpt
+	case Cost:
```

Comment:
So we support setting cost with key like `cost.update_settings`, but we only support getting the whole cost map?

Reply:
Yes, for now it is this way. Basically that's how I found the output for cost in previous bit of codes. but I'm going to have a discussion with @dabasov about it, whether map or cost.* should be the way we send output back for get requests.

Reply:
output updated to cost.function_name, for all the values in cost map.